Tests/Parser/Client/fixtures/browser.yml

```diff
@@ -7937,3 +7937,213 @@
     family: Chrome
   headers:
     Sec-CH-UA: '" Not A;Brand";v="99.0.0.0", "Chromium";v="2024.01"'
+-
+  user_agent: ArcMobile2/1 CFNetwork/1494.0.5 Darwin/23.4.0
+  client:
+    type: browser
+    name: Arc
+    version: ""
+    engine: WebKit
+    engine_version: ""
+    family:
+-
+  user_agent: Lilo/66 CFNetwork/978.0.7 Darwin/18.7.0
+  client:
+    type: browser
+    name: Lilo
+    version: ""
+    engine: WebKit
+    engine_version: ""
+    family: Chrome
+-
+  user_agent: com.airfind.browser/2.0.5 (Linux; U; Android 8.1.0; es-us; AX751+ Build/O11019; Bmobile) 480X854 Bmobile AX751+ AppleWebKit/537.36 (KHTML, like Gecko)
+  client:
+    type: browser
+    name: Airfind Secure Browser
+    version: 2.0.5
+    engine: Blink
+    engine_version: ""
+    family: Chrome
+  headers:
+    http-x-requested-with: com.airfind.browser
+-
+  user_agent: 'Dalvik/2.1.0 (Linux; U; Android 10; ZTE 8010 Build/QP1A.190711.020) AppleWebKit [HB/29]'
+  client:
+    type: browser
+    name: SecureX
+    version: ""
+    engine: WebKit
+    engine_version: ""
+    family: Android Browser
+  headers:
+    http-x-requested-with: com.securex.browser
+-
+  user_agent: DoCoMo/2.0 SH901iC(c100;TB;W24H12)
+  client:
+    type: browser
+    name: DoCoMo
+    version: "2.0"
+    engine: ""
+    engine_version: ""
+    family:
+-
+  user_agent: Mozilla/5.0 (Linux; Android 7.0; LG-M250 Build/NRD90U; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/62.0.3202.84 Mobile Safari/537.36
+  client:
+    type: browser
+    name: Opera
+    version: ""
+    engine: Blink
+    engine_version: 62.0.3202.84
+    family: Opera
+  headers:
+    http-x-requested-with: com.opera.browser.beta
+-
+  user_agent: Mozilla/5.0 (Linux; Android 8.1.0; SM-G610M Build/M1AJQ; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/98.0.4758.101 Mobile Safari/537.36
+  client:
+    type: browser
+    name: Yandex Browser
+    version: ""
+    engine: Blink
+    engine_version: 98.0.4758.101
+    family: Chrome
+  headers:
+    http-x-requested-with: com.yandex.browser.beta
+-
+  user_agent: Mozilla/5.0 (Linux; Android 11; SM-A715F Build/RP1A.200720.012; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/100.0.4896.88 Mobile Safari/537.36
+  client:
+    type: browser
+    name: Yandex Browser
+    version: ""
+    engine: Blink
+    engine_version: 100.0.4896.88
+    family: Chrome
+  headers:
+    http-x-requested-with: com.yandex.browser.alpha
+-
+  user_agent: Mozilla/5.0 (Linux; Android 11; SM-S506DL Build/RP1A.200720.012; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/101.0.4951.61 Mobile Safari/537.36
+  client:
+    type: browser
+    name: Samsung Browser
```

Comment:
Should we not differentiate this one (`Samsung Browser Beta`), since this is beta version, which is clear both from ID and name in PlayStore? I mean, since Samsung released this as a completely separate entity instead of using "beta program" or whatever it's called

Reply:
I've had the same idea a few years ago, but hasn't been approved. See https://github.com/matomo-org/device-detector/pull/6073#issuecomment-544256158